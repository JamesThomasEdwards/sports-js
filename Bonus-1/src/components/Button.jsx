// React;
import React from 'react';

const Button = (props) => {
    return (
        <button
            onClick={props.handeler}>
            {props.buttonTitles}
        </button>
    )
}

export default Button