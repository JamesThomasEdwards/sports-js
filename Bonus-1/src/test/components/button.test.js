import React from 'react';
import { shallow } from 'enzyme';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Button from '../../components/Button.jsx';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render Button component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Button />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});