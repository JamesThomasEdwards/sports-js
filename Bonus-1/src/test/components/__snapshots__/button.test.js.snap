// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Button component correctly 1`] = `
<div
  className="button-container"
>
  <button />
</div>
`;
